web/.env.local.example

```
@@ -0,0 +1,9 @@
+# Copy this to .env.local for local development
+# cp .env.local.example .env.local
+
+DISCORD_CLIENT_ID=
+DISCORD_CLIENT_SECRET=
+NEXTAUTH_SECRET=change-me-in-production
+NEXTAUTH_URL=http://localhost:3000
+BOT_API_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BOT_API_URL key should go before the DISCORD_CLIENT_ID key

(UnorderedKey)